### PR TITLE
added new item to settings for matomo

### PR DESCRIPTION
**Pull request by LianaHus**
fix https://github.com/ethereum/remix-project/issues/887

### CODE REVIEW

File: apps/remix-ide/src/app/tabs/settings-tab.js [hunk 1]
```diff
@@ -103,6 +105,17 @@ module.exports = class SettingsTab extends ViewPlugin {
     if (this.config.get('settings/personal-mode')) this._view.personal.setAttribute('checked', '')
     elementStateChanged(self._view.personalLabel, !this.config.get('settings/personal-mode'))
 
+    this._view.useMatomoAnalytics = yo`<input onchange=${onchangeMatomoAnalytics} id="settingsMatomoAnalytics" type="checkbox" class="custom-control-input">`
+    this._view.useMatomoAnalyticsMode = yo`<i class="${css.icon} fas fa-exclamation-triangle text-warning" aria-hidden="true"></i>`
```
Comment (yann300) [inline apps/remix-ide/src/app/tabs/settings-tab.js:109]: `this._view.useMatomoAnalyticsMode` is not defined in the constructor

File: apps/remix-ide/src/app/tabs/settings-tab.js [hunk 2]
```diff
@@ -186,6 +207,13 @@ module.exports = class SettingsTab extends ViewPlugin {
       self.config.set('settings/personal-mode', !isChecked)
       elementStateChanged(self._view.personalLabel, isChecked)
     }
+    function onchangeMatomoAnalytics () {
+      const isChecked = self.config.get('settings/matomo-analytics')
+
+      self.config.set('settings/matomo-analytics', !isChecked)
```
Comment (yann300) [inline apps/remix-ide/src/app/tabs/settings-tab.js:213]: you set 'settings/matomo-analytics' ok , but where do you actually say matomo to stop?

Comment (LianaHus) [reply apps/remix-ide/src/app/tabs/settings-tab.js:213]: I disable this from UI. I'm not sure it is possible. I will work on this once we have it working

File: apps/remix-ide/src/app/components/plugin-manager-component.js [hunk 3]
```diff
@@ -101,6 +102,16 @@ class PluginManagerComponent extends ViewPlugin {
     return this.appManager.actives.includes(name)
   }
 
+  activate (name) {
+    this.appManager.activatePlugin(profile.name)
+    _paq.push(['trackEvent', 'manager', 'activate', name])
+  }
+
+  deactivate (name) {
+    this.appManager.deactivatePlugin(profile.name)
```
Comment (yann300) [inline apps/remix-ide/src/app/components/plugin-manager-component.js:111]: that is not declared `profile.name`

**Review by yann300 — APPROVED** [review]

**Pull request merged**